app.yaml

```diff
@@ -53,6 +53,12 @@ handlers:
   http_headers:
     Cache-Control: 'public, max-age=2592000'
     Vary: Accept-Encoding
+- url: /third_party/generated
+  static_dir: third_party/generated
+  secure: always
+  http_headers:
+    Cache-Control: 'public, max-age=2592000'
```

Comment:
I think maybe we ought to make this no-cache for now until we figure out a system to break the cache when there's a new release. Could we do that and add a TODO?